File: frontend/lib/models/sablonas.dart
class Sablonas {
  final int id;
  final String pavadinimas;

  const Sablonas({required this.id, required this.pavadinimas});

  factory Sablonas.fromJson(Map<String, dynamic> j) => Sablonas(
    id: (j['id'] ?? 0) as int,
    pavadinimas: (j['pavadinimas'] ?? j['Pavadinimas'] ?? '') as String,
  );

  Map<String, dynamic> toJson() => {'id': id, 'pavadinimas': pavadinimas};
}

File: Backend/Models/Sablonas.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    [Table("sablonas")]
    public class Sablonas
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Required]
        [Column("pavadinimas")]
        public string Pavadinimas { get; set; } = null!;

        [JsonIgnore]
        public ICollection<SablonasTestas> Testai { get; set; } = new List<SablonasTestas>();
    }
}
